submodules/tags/README.md

```diff
@@ -0,0 +1,60 @@
+# Tags
+
+Module to create tags
+
+<!-- markdownlint-disable MD013 MD033 -->
+## Input Variables
+
+| Name            | Type                                                                        | Default Value | Example                                                                               |
+| --------------- | --------------------------------------------------------------------------- | ------------- | ------------------------------------------------------------------------------------- |
+| user_tags       | object({ <br />team = string<br /> purpose = string<br /> owner = string<br /> }) |               | {<br />  team= "devops"<br /> purpose= "elk module test"<br /> owner= "Samir"<br /> } |
+| octopus_tags    | object({ <br />project = string<br /> space = string<br />  })                    |               | {<br />  project = "actions-test"<br /> space   = "Default"<br /> }                   |
+| name            | string                                                                      |               | "Test"                                                                                |
+| octopus_project | string                                                                      |               | "actions-test"                                                                        |
+| octopus_space   | string                                                                      |               | "Default"                                                                             |
+<!-- markdownlint-enable MD013 MD033 -->
+
+## Example .tf file module reference
+
+```bash
+  module "tags" {
+    source = "github.com/variant-inc/lazy-terraform//submodules/tags?ref=v1"
+
+    octopus_tags = {
+      team = "devops"
+      purpose = "elk module test"
+      owner = "Samir"
+    }
+    octopus = {
```

Comment:
This need to be octopus_tags as per your varibales.tf

Reply:
Done